src/main/java/com/thevoxelbox/voxelsniper/VoxelSniperPlugin.java

```diff
@@ -83,6 +85,8 @@ public void onEnable() {
         PaperLib.suggestPaper(this);
         currentVersionTitle = getDescription().getVersion().split("-")[0];
         currentVersion = Double.parseDouble(currentVersionTitle.replaceFirst("\\.", ""));
+        // Don't register task if update check is unwanted
+        if (!voxelSniperConfig.isUpdateCheckerEnabled()) return;
```

Comment:
We don't impose a specific format, but something like this with explicit brackets may be better to fit to FAVS "conventions".
```java
if (!voxelSniperConfig.isUpdateCheckerEnabled()) {
    return;
}
```